libs/core-ui/src/lib/Highchart/bubbleChart/buildScatterTemplate.ts

```diff
@@ -16,21 +16,45 @@ export interface ICustomData {
 export function buildScatterTemplate(
```

Comment:
could you maybe rename the function buildScatterTemplate? There are other functions in the repo with same name.

Reply:
Thanks Gaurav, will update the name in the coming PR.